wsi_deid/web_client/views/HierarchyWidget.js

```diff
@@ -50,10 +57,10 @@ function performAction(action) {
         if (resp.action === 'export' || resp.action === 'exportall') {
             [
                 ['finished', 'exported'],
-                ['present', 'previously exported'],
-                ['different', 'already present but different.  Remove images from the export directory and select Export again'],
-                ['quarantined', 'currently quarantined.  Only files in "Approved" workflow stage are transferred to Export folder'],
-                ['rejected', 'with rejected status.  Only files in "Approved" workflow stage are transferred to Export folder']
+                ['present', 'previously exported and already exist in export folder'],
```

Comment:
maybe `already exists(s)` if the number is variable. it's hard for me to tell from this fragment what the subject/verb is.